Sources/RegexBuilder/Variadics.swift

```diff
@@ -571,7 +571,11 @@ extension RegexComponentBuilder {
     accumulated: R0, next: R1
   ) -> Regex<Substring> where R0.RegexOutput == W0  {
     let factory = makeFactory()
-    return factory.accumulate(accumulated, next)
+    if #available(macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, *) {
```

Comment:
Does `if #available(SwiftStdlib 5.8, *)` work?

Reply:
No, that isn't allowed in an always-emit function.